controllers/operatorpolicy_status.go

```diff
@@ -811,6 +838,12 @@ func catalogSourceFindCond(isUnhealthy bool, isMissing bool, name string) metav1
 		message = "CatalogSource '" + name + "' was not found"
 	}
 
+	// Only override if condition evaluated to NonCompliant
+	if status == metav1.ConditionTrue && complianceConfig == "Compliant" {
+		status = metav1.ConditionFalse
+		message += " - ignore, compliance overridden by complianceConfig"
```

Comment:
I don't think this message is required since the default is "Compliant" now.

Reply:
Correct me if I'm wrong but I think the resolution for this is similar to above conversations (for installPlanUpgradeCond)? Would it be fine if I modified message to say something like `(policy compliance is not impacted due to spec.complianceConfig.catalogSourceUnhealthy)`?